api/apps/oura/get.ts

```diff
@@ -0,0 +1,34 @@
+import createAPIGatewayProxyHandler from "package/backend/createAPIGatewayProxyHandler";
+
+// https://cloud.ouraring.com/v2/docs
+
+type ParamProps = {
+  dataType: string;
+  date: string;
+  startDate: string;
+  endDate: string;
+  token: string;
+};
+const logic = async ({
+  dataType,
+  date, // TODO - need more data
+  endDate,
+  startDate,
```

Comment:
Previous version had start/end as same date, but currently I need different dates to return data.  But I only have a few hours of data.  Waiting for more to test.

Reply:
Inconsistent.

`daily_sleep` and `daily_readiness` gave a response with a single date (2024-01-27)
`sleep` and `daily_activity` responded with null for a single date, but proper data when two dates were used (2024-01-27 and 2024-01-28)